generator/integration_tests/golden/internal/golden_kitchen_sink_option_defaults.gcpcxx.pb.cc

```diff
@@ -31,6 +31,10 @@ namespace cloud {
 namespace golden_internal {
 inline namespace GOOGLE_CLOUD_CPP_GENERATED_NS {
 
+namespace {
+auto constexpr kBackoffScaling = 2.0;
```

Comment:
This should be moved inside the function since that's the only place it's used.

Reply:
If this service has any longrunning operation methods, (compare to golden_thing_admin_option_defaults.gcpcxx.pb.cc), it is used in more than once place.

Now, I can add logic to vary the placement of this definition depending on the presence of LRO methods, but is it worth it?